src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
Ok, any suggestions on how? Should I place this in it's own header `cloudsettings.h`, or duplicate the definition, or include `clouds.h` in `remoteplayer.h` (a bit overkill)?